dataservice/api/family/resources.py

```diff
@@ -28,7 +29,17 @@ def get(self, after, limit):
             resource:
               Family
         """
-        q = Family.query
+        q = Family.query.options(load_only('kf_id'))
+
+        # Filter by study
+        from dataservice.api.participant.models import Participant
+        study_id = request.args.get('study_id')
+        if study_id:
+            q = (q.join(Family.participants)
+                 .options(Load(Participant).load_only('kf_id', 'study_id'))
+                 .filter(Participant.study_id == study_id)
+                 .distinct(Family.kf_id)
+                 .order_by(Family.kf_id))
```

Comment:
I have to include the order_by because of the call to distinct. If I don't, PG throws an exception. Its because the DISTINCT ON expression needs to know which row to pick out of the ones that matched on Family.kf_id.